jck/runtime.api/playlist.xml

```diff
@@ -41,6 +41,18 @@
 		<groups>
 			<group>jck</group>
 		</groups>
+		<disabled>
+			<comment>Disabled due to TKG/issues/123. Awaiting support in TKG to specify a path for parallelList.mk</comment>
+			<impl>ibm</impl>
+		</disabled>
+		<disabled>
+			<comment>Disabled due to TKG/issues/123. Awaiting support in TKG to specify a path for parallelList.mk</comment>
+			<impl>openj9</impl>
+		</disabled>
+		<disabled>
+			<comment>Disabled due to TKG/issues/123. Awaiting support in TKG to specify a path for parallelList.mk</comment>
+			<impl>hotspot</impl>
```

Comment:
If we exclude this for all impl, then we do not need to specify `<impl>`

Reply:
Also, please use the new disable format https://github.com/eclipse-openj9/openj9/blob/master/test/docs/OpenJ9TestUserGuide.md#exclude-test-against-multiple-criteria

Reply:
Updated.